packages/tables/README.md

```diff
@@ -0,0 +1,56 @@
+# @zendeskgarden/react-tables [![npm version](https://img.shields.io/npm/v/@zendeskgarden/react-tables.svg?style=flat-square)](https://www.npmjs.com/package/@zendeskgarden/react-tables)
+
+This package includes components relating to tables in the
+[Garden Design System](https://zendeskgarden.github.io/).
+
+## Installation
+
+```sh
+npm install @zendeskgarden/react-tables
+
+# Peer Dependencies - Also Required
+npm install react react-dom prop-types styled-components @zendeskgarden/react-theming
+```
+
+## Usage
+
+```jsx static
+/**
+ * Include tables styling at the root of your application
+ */
+import '@zendeskgarden/react-tables/dist/styles.css';
+
+import { ThemeProvider } from '@zendeskgarden/react-theming';
+import { Table, Caption, Head, Body, Row, HeaderCell, Cell } from '@zendeskgarden/react-tables';
+
+/**
+ * Place a `ThemeProvider` at the root of your React application
+ */
+<ThemeProvider>
+  <Table>
+    <Caption>Your Unsolved Tickets</Caption>
+    <Head>
+      <Row header>
+        <HeaderCell>Subject</HeaderCell>
+        <HeaderCell>Requester</HeaderCell>
+        <HeaderCell>Requested</HeaderCell>
+        <HeaderCell>Type</HeaderCell>
```

Comment:
It'd be good to add the `scope` attribute here to show best practice with table a11y. 

```jsx
<Row header>
  <HeaderCell scope="col">Subject</HeaderCell>
  <HeaderCell scope="col">Requester</HeaderCell>
  <HeaderCell scope="col">Requested</HeaderCell>
  <HeaderCell scope="col">Type</HeaderCell>
</Row>
```

Reply:
TIL https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Advanced#Tables_for_visually_impaired_users

Didn't realize how useful `scope` was. I'll add this to all of the examples.